lib/arborist/reify.js

```diff
@@ -525,6 +530,8 @@ module.exports = cls => class Reifier extends cls {
         }
         await this[_checkBins](node)
         await this[_extractOrLink](node)
+        if (node.isWorkspace && this[_saveWorkspaceRoot])
+          await this[_writeWorkspaceRootConfig](node)
```

Comment:
this shouldn't be written if the workspace root already exists in the npmrc file - only if it's missing (either the file, or that config value)